google/cloud/speech/operation.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Long running operation representation for Google Speech API"""
+
+from google.cloud._helpers import _rfc3339_to_datetime
+
+
+class Operation(object):
```

Comment:
I thought they would diverge more than they did. I'll subclass it in the updates.